src/OpenTelemetry.Exporter.Jaeger/README.md

```diff
@@ -45,6 +44,38 @@ See the
 [`TestJaegerExporter.cs`](../../examples/Console/TestJaegerExporter.cs)
 for an example of how to use the exporter.
 
+## Resources and Process Tags
+
+The attributes of a [Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
+ are represented in Jaeger as `ProcessTags` that associate with an
+exported span. The exception is the [required](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#semantic-attributes-with-sdk-provided-default-value)
+`service.name` attribute which accompanies each exported span as shown below.
+
+Example configuration of resource attributes and attachment to `TracerProvider`
+with `JaegerExporter`:
+
+```csharp
+ List<KeyValuePair<string, object>> resourceAttrib = new() {
+    new KeyValuePair<string, object>("attribute1", "ABC"),
+    new KeyValuePair<string, object>("attribute2", 123),
+    new KeyValuePair<string, object>("service.name", ".NET OTel Service")
+};
+
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .SetResourceBuilder(
+        ResourceBuilder.CreateDefault().AddAttributes(resourceAttrib))
+    .AddSource("mysource")
+    .AddJaegerExporter()
+    .Build();
+```
+
+When the source configured for above emits two generic, nested activities
+the resulting telemetry is captured on Jaeger as shown below. The
+`service.name` attribute is not shown with the rest of the attributes as
+`ProcessTags`, but rather separately as a header for each span representing the
+service it originated from.
+![Image of Jaeger UI showing presence of process tags](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.Jaeger/jaeger-resource.PNG)
```

Comment:
CI Pipeline wants a direct URL like this rather than relative "./jaeger-resource.PNG" link. It won't show right now but once merged the image should show up since it's a link to main. Please check the image below in files.